Reviews/153 - Uniswap Clone for Casper/1 - Contract Port 4 Engineers + 2 QA Engineers/README.md

```diff
@@ -0,0 +1,162 @@
+We are able to deploy the uniswap contract and get it working with a compatible erc20 | PASS
+
+# Unit / Automated Testing
+
+Working tests are not required for this review to pass the acceptance criteria. 
+Despite this, the OP is well on their way to having adaquate tests.
+
+There are Positive & Negative Test Cases. 
+They are insufficent for the end of project,
+but for the current state of the project they are sufficient. 
+
+
+Requirement | Finding
+------------ | -------------
+Unit Tests - Sufficeint | PASS
+
+# Documentation
+
+The documentation for this project is excellent for this stage of the process.
+They have a table of contents with links and type descriptions. 
+They far are ahead of their milestones in this capacity.
+Excellent. 
+
+### Code Documentation
+
+Next to the fantastic supporting documentation the inline documentation is exceedingly adaquate.
+But again, for this stage of the process, this is more than sufficient.
+The reviewer does urges the OP to include much more inline documentation in the source code. 
+This would be very helpful for untrained newcomers whom actually want to read the soruce. 
+That being said, it should also be noted that the OP has left plenty of space near each 
+method to do so and very well may have intended to expand the documentation intime.
+In all, a promising display.   
+
+Requirement | Finding
+------------ | -------------
+Code Documented | PASS
+
+
+### Project Documentation
+
+Project Documentation was acceptable for this stage of the project. 
+
+Requirement | Finding
+------------ | -------------
+Usage Documented | PASS
+Example Documented | PASS
+
+## Overall Conclusion on Documentation
+
+An impressive display for this stage. 
+
+# Open Source Practices
+
+## Licenses
+
+The Project is released under the Apache License 2.0
+
+Requirement | Finding
+------------ | -------------
+OSI-approved open source software license | PASS
+
+
+## Contribution Policies
+
+The current state of the code is in private repositories. 
+As this code is sensitive in nature at the moment, it is the opinion of this reviewer that this is 
+appropriate, given the circumstances. If a malicious actor was to gain access to this code and set 
+up a DEX, the reputation of the Casper Ecosystem could be affected. 
+
+The OP has indicated that he is willing to show the 
+repository to any individual that may ask. 
+In light og the high-profile nature of the project, 
+this seems reasonable and nessecary and this reveiwer 
+urges other policing coleagues to consider this and 
+waive the requirement. 
+
+
+Requirement | Finding
+------------ | -------------
+OSS contribution best practices | PASS
+
+
+# Coding Standards
+
+Acceptable. 
+
+## General Observations
+
+# Final Conclusion
+
+It has not been mentioned yet, but two other reviews have been done: 
+ { Quantstamp's 1st Report, https://github.com/atomizable/dxd_codereview/blob/main/153%20-%20Uniswap%20Clone%20for%20Casper/Uniswap%20-%20Report.pdf }
```

Comment:
You should use a relative link to the artifact, as this link will stop working once you close the review and delete the branch